node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/containerdata/ContainerData.java

```diff
@@ -0,0 +1,90 @@
+package com.yahoo.vespa.hosted.node.admin.containerdata;
+
+import com.yahoo.io.IOUtils;
+import com.yahoo.log.LogLevel;
+import com.yahoo.text.Utf8;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+/**
+ * This class can be used for adding files that should be accessible in a container's file system after
+ * the container has been started.
+ * <p>
+ * Files that are added will be copied into the destination path on the host.
+ * The entry point for a Docker image will take care of copying
+ * everything into it's right place (ATM only done for config server nodes)
+ * Note: Creating a new instance of this will cleanup all old data in the destination path
```

Comment:
Consider making a named static method for creating ContainerData to make this clearer (createCleanContainerData() or similar), since modifying the filesystem in a constructor may come as a surprise.